scripts/automation/s3-test/run_testcases.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/sh
+set +e
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+# Help function which stdouts possible arguments for this script 
+_help_function(){
```

Comment:
can you use HEREDOC for usage function.
Refer beow - 
https://github.com/Seagate/cortx-re/blob/96d3dc34f0ff3f8107c908c5d67f8181b5e372d9/solutions/kubernetes/cluster-functions.sh#L30

Reply:
Changes help function please review once.